code/game/objects/items.dm

```diff
@@ -756,7 +758,16 @@
 	if(!delay && !tool_start_check(user, amount))
 		return
 
+	var/skill_bonus = 1
+	//default check for item
+	if(skill_checks)
+		skill_bonus = apply_skill_bonus(user, 1, skill_checks)
+	//in case item have no defined default required_skill or we need to check other skills e.g. check crowbar for surgery
+	if(required_skills)
```

Comment:
Переименовал параметр что принимает на  required_skills_override и заменил if на тот что ты выше написал, то есть
```
//in case item have no defined default required_skill or we need to check other skills e.g. check crowbar for surgery
if(required_skills_override)
	skill_bonus = apply_skill_bonus(user, 1, required_skills_override)
else if(required_skills) //default check for item
	skill_bonus = apply_skill_bonus(user, 1, required_skills)
```